.github/workflows/build_develop.yml

```
@@ -50,12 +59,62 @@ jobs:
           name: webapp
           path: dist/develop.tar.gz
           retention-days: 1
+          
+      - name: Extract webapp
+        run: |
+          mkdir _deploy
+          tar xf dist/develop.tar.gz -C _deploy --strip-components=1
+          
+      - name: Copy config
+        run: cp element.io/develop/config.json _deploy/config.json
+
+      - name: Populate 404.html
+        run: echo "404 Not Found" > _deploy/404.html
+
+      - name: Populate _headers
+        run: cp .github/cfp_headers _deploy/_headers
+
+      - name: Populate _redirects
+        run: |
+          echo "/develop.tar.gz $R2_PUBLIC_URL/develop.tar.gz 301" >> _deploy/_redirects
+          for bundle in $(aws s3 ls s3://$R2_BUCKET/bundles/ --endpoint-url $R2_URL --region=auto | awk '{print $2}'); do
+            echo "/bundles/${bundle}* $R2_PUBLIC_URL/bundles/${bundle}:splat 301" >> _deploy/_redirects
+          done
```

If this were a bash script, I'd do:

```suggestion
          {
            echo "/develop.tar.gz $R2_PUBLIC_URL/develop.tar.gz 301"
            for bundle in $(aws s3 ls s3://$R2_BUCKET/bundles/ --endpoint-url $R2_URL --region=auto | awk '{print $2}'); do
              echo "/bundles/${bundle}* $R2_PUBLIC_URL/bundles/${bundle}:splat 301"
            done
          }  > _deploy/_redirects
```

Not sure if that works right when it's embedded in GHA.

---

This does indeed seem to work, switched to `tee` to inline the cat too